src/run/formlyNgModelAttrsManipulator.js

```diff
@@ -18,7 +18,7 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate) {
       return template;
     }
     el.innerHTML = template;
-    var modelNodes = el.querySelectorAll('[ng-model], [data-ng-model]');
+    var modelNodes = el.querySelectorAll('[ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing), [data-ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing)');
```

Comment:
Also, this line is quite long (and I've updated the pre-commit hook so you wont be able to do this anymore), perhaps you could use the [ES6 template string expression interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings#Expression_interpolation) and pull parts of this out into variables that you then use to construct the `query` that's passed to the `querySelectorAll`.